io.github.KangLin.SerialPortAssistant.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "io.github.KangLin.SerialPortAssistant",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "SerialPortAssistant",
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=all",
+        "--filesystem=xdg-documents"
+    ],
+    "modules": [
+        {
+            "name": "cmark",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DBUILD_TESTING=OFF"
+            ],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/commonmark/cmark/archive/refs/tags/0.31.1.tar.gz",
+                    "sha256": "3da93db5469c30588cfeb283d9d62edfc6ded9eb0edc10a4f5bbfb7d722ea802"
+                }
+            ]
+        },
+        {
+            "name": "RabbitCommon",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+		"-DBUILD_MimeTypeBrowser=OFF",
+		"-DBUILD_APP=OFF",
+		"-DBUILD_TEST=OFF"
+            ],
+            "sources": [
```

Comment:
![image](https://github.com/user-attachments/assets/d4ad5253-636c-4f88-bb61-de04c927a56c)

indentation is broken here